nodes/core/logic/15-change.js

```diff
@@ -166,12 +166,10 @@ module.exports = function(RED) {
                             try {
                                 fromRE = new RegExp(fromRE, "g");
                             } catch (e) {
-                                reject(new Error(RED._("change.errors.invalid-from",{error:e.message})));
-                                return;
+                                return Promise.reject(new Error(RED._("change.errors.invalid-from",{error:e.message})));
```

Comment:
This function returns a promise that needs rejecting in this scenario. Why does calling `reject` not work but returning a rejected pomise does work? Would like to understand that before merging. I'm away from my laptop today. Will look properly tonight

Reply:
These codes are in `then` function scope, so `reject` cannot be found here.

Reply:
Ahhh yes. Didn't spot that when reviewing on my phone screen :+1: